app/config/config_dev.yml

```diff
@@ -4,4 +4,13 @@ imports:
 doctrine:
     dbal:
         driver: pdo_sqlite
-        path: "%kernel.root_dir%/../var/user-lifecycle.sqlite"
\ No newline at end of file
+        path: "%kernel.root_dir%/../var/user-lifecycle.sqlite"
+
+monolog:
+    handlers:
+        main:
+            type: stream
+            path: '%kernel.logs_dir%/%kernel.environment%.log'
+            level: debug
+        console:
```

Comment:
The `console` entry in the `config_dev.yml` is not adding anything right? In my functional test I could remove it without any issues.